api/src/main/java/gov/va/vro/api/resources/MasResource.java

```diff
@@ -61,4 +62,36 @@ ResponseEntity<MasClaimResponse> notifyAutomatedClaimDetails(
           @Valid
           @RequestBody
           MasClaimDetailsPayload request);
+
+  @Operation(
+      summary = "MAS Exam Ordering Status",
+      description = "Request Ordering Status for an exam")
+  @PostMapping("/examOrderingStatus")
+  @ResponseStatus(HttpStatus.CREATED)
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "201", description = "Successful Request"),
+        @ApiResponse(
+            responseCode = "401",
+            description = "Unauthorized",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "500",
+            description = "Data Access Server Error",
+            content = @Content(schema = @Schema(hidden = true))),
+        @ApiResponse(
+            responseCode = "404",
```

Comment:
Are we going to have 404 response really?

Reply:
I don't know. @rajesh-Krishnasamy ?